source/presentation/4.0/index.md

```diff
@@ -74,25 +74,24 @@ __Previous Version:__ [3.0][prezi30]
 
 The purpose of the IIIF Presentation API specification is to provide a [model](model) and JSON serialization format of that model.
 
-It provides a document format - the IIIF Manifest - for cultural heritage organizations (and anyone else) to present objects in a standardized, interoperable fashion. This allows compatible software such as viewers and annotation tools to load and present complex digital objects on the web from thousands of different providers.
+It provides a document format---the IIIF Manifest---for cultural heritage organizations (and anyone else) to present objects in a standardized, interoperable way. This allows compatible software such as viewers and annotation tools to load and present complex digital objects on the web from thousands of different providers.
```

Comment:
Does this work for em-dashes in the markdown parser we use? Just checking.

Reply:
Answered my own question, I guess it does! https://preview.iiif.io/api/prezi-4-dawn/presentation/4.0/